capa/render/utils.py

```diff
@@ -61,6 +61,21 @@ def capability_rules(doc: rd.ResultDocument) -> Iterator[rd.RuleMatches]:
         yield rule
 
 
+def maec_rules(doc: rd.ResultDocument) -> Iterator[rd.RuleMatches]:
+    """enumerate the rules in (namespace, name) order that are 'maec' rules."""
+    for _, _, rule in sorted((rule.meta.namespace or "", rule.meta.name, rule) for rule in doc.rules.values()):
```

Comment:
why in this order?

Reply:
also: consider factoring the sorting into its out subroutine since it's also used elsewhere.

Reply:
@mr-tz, I simply re-used the implementation from `capability_rules`. Do you suggest using a different sorting order?

Reply:
I think it makes sense for capability rules but the order isn't relevant here.